docs/getting-started/index.mdx

```diff
@@ -6,174 +6,377 @@ pagination_next: http/index
+		
+		Find all the options on our
+			<a href="https://download.ngrok.com/">downloads page</a>.
+		
+    The ngrok agent is a zero-dependency CLI program that runs on all major
+    operating systems. Test that you installed it correctly by running the
+    following command in your terminal and confirm that ngrok prints its help text.
 
-```bash
-brew install ngrok/ngrok/ngrok
-```
+    ```bash
+    ngrok help
+    ```
 
-</TabItem>
-<TabItem value="linux" label="Debian Linux">
+  </li>
+  <li>
+		<span className="text-strong text-xl font-bold">A local application or API</span>
 
-```bash
-curl -s https://ngrok-agent.s3.amazonaws.com/ngrok.asc | \
-  sudo gpg --dearmor -o /etc/apt/keyrings/ngrok.gpg && \
-  echo "deb [signed-by=/etc/apt/keyrings/ngrok.gpg] https://ngrok-agent.s3.amazonaws.com buster main" | \
-  sudo tee /etc/apt/sources.list.d/ngrok.list && \
-  sudo apt update && sudo apt install ngrok
-```
+    Accessible on a hostname and port like `localhost:8000`.
 
-</TabItem>
-<TabItem value="windows" label="Windows">
+  </li>
+</ol>
 
-```bash
-choco install ngrok
-```
-
-Run this in an Administrator Command Prompt.
+## Step 1: Connect your account
 
-</TabItem>
-</Tabs>
+Next, connect your ngrok agent to your ngrok account.
 
-Or [download the ngrok agent from our Download
-page](https://download.ngrok.com) if you can't use one of the options above.
+Head over to your [ngrok
+dashboard](https://dashboard.ngrok.com/get-started/your-authtoken) and click
+**Copy** to get your authtoken.
 
-Going to the [Setup & Installation page in the dashboard](https://dashboard.ngrok.com/get-started/setup) will also provide installation directions and commands specific to your platform.
-
-The ngrok agent is a zero-dependency CLI program that runs on all major
-operating systems. Test that you installed it correctly by running the
-following command in your terminal and confirm that ngrok prints its help text.
+Run the following command in your terminal to add the authtoken and connect the
+ngrok agent to your account.
 
 ```bash
-ngrok help
+ngrok config add-authtoken <NGROK_AUTHTOKEN>
 ```
 
-## Step 2: Connect your account
+The [Setup & Installation page in the
+dashboard](https://dashboard.ngrok.com/get-started/setup) also provides a
+configuration command specific to your account.
 
-Next, connect your ngrok agent to your ngrok account. If you haven't already,
-[sign up for an ngrok account](https://dashboard.ngrok.com). Copy your [ngrok
-authtoken](https://dashboard.ngrok.com/get-started/your-authtoken) from your
-ngrok dashboard.
+## Step 2: Put your app or API online
 
-Run the following command in your terminal to install the authtoken and connect
-the ngrok agent to your account.
+Start the ngrok agent with the following command, replacing `8080` with a
+different port based on where your service listens for traffic.
 
 ```bash
-ngrok config add-authtoken <TOKEN>
+ngrok http 8080
```

Comment:
earlier, we told them to have something running on port 8000, now we are using port 8080. might cause confusion.